pkg/storaged/stratis-details.jsx

```diff
@@ -703,11 +876,21 @@ export function start_pool(client, uuid, show_devs) {
                         unlock_with_keydesc(key_desc);
                 }));
     }
+
+    if (!key_desc && !clevis_info) {
+        // Not an encrypted pool, just start it
+        return start();
+    } else if (key_desc && clevis_info) {
+        return start("clevis").catch(unlock_with_keyring);
+    } else if (!key_desc && clevis_info) {
+        return start("clevis");
+    } else if (key_desc && !clevis_info) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230707-100043-8d39ae7d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)